config/ignition_fortress_ubuntu_jammy.yaml

```
@@ -68,12 +68,12 @@ filter_formula: "\
  Package (= libignition-physics5-tpe-dev) |\
  Package (= libignition-physics5-tpelib) |\
  Package (= libignition-physics5-tpelib-dev) \
-), ($Version (% 5.1.0+osrf-2*) | $Version (% 5.1.0+osrf-3*)) |\
+), ($Version (% 5.2.0-1*) | $Version (% 5.1.0+osrf-3*)) |\
```

Should we remove the alternative version?

```suggestion
), $Version (% 5.2.0-1*) |\
```